src/parts/DefaultSection.js

```diff
@@ -66,7 +67,8 @@ const DefaultSection = {
               class: {
                 "autosuggest__results_item-highlighted":
                   this.getItemIndex(key) == this.currentIndex,
-                autosuggest__results_item: true
+                autosuggest__results_item: true,
+                [this.section.listItemClassName]: this.section.listItemClassName
```

Comment:
I don't mind adding this, but instead of restricting it to class, we should instead bind any attributes using v-bind which can accept an object.